src/mock_server/builder.rs

```diff
@@ -76,6 +87,30 @@ impl MockServerBuilder {
         self
     }
 
+    /// By default, when printing requests the size of the body that can be
+    /// printed is limited.
+    ///
+    /// This may want to be changed if working with services with very large
+    /// bodies, or when printing wiremock output to a file where size matters
+    /// less than in a terminal window. You can configure this limit with
+    /// `MockServerBuilder::body_print_limit`.
+    pub fn body_print_limit(mut self, limit: usize) -> Self {
```

Comment:
Let's do something slightly different: introduce a `BodyPrintLimit` enum with two variants, `Limited(usize)` and `Unlimited`. We should use the same representation throughout the internal and the external API.
Then we can avoid `disable_body_print_limit` and manage everything in a single place.